module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -14,11 +14,13 @@ If you have a different version of R or other R packages, the documentation may
 **Table of Contents**
 
 - [Base `R`](#base-r)
-- [`SingleCellExperiment`, `scran`, and `scater`](#singlecellexperiment-scran-and-scater)
+- [`SingleCellExperiment`, `DropletUtils`, `scran`, and `scater`](#singlecellexperiment-dropletutils-scran-and-scater)
 - [`purrr`](#purrr)
 - [`stringr`](#stringr)
 - [`alevinQC`, `colorblindr`, `Rtsne`, `tibble`](#alevinqc-colorblindr-rtsne-tibble)
```

Comment:
Noting that we don't use `colorblindr` anymore and probably not `Rtsne` either. This table also has some odd contents, as it includes `S4Vectors::DataFrame`, which should probably go with SCE? And I would put alevinQC with Salmon maybe?